ui/src/app/components/runtime-configuration-panel.spec.tsx

```diff
@@ -1748,4 +1807,25 @@ describe('RuntimeConfigurationPanel', () => {
     wrapper.update();
     expect(wrapper.text()).toContain('MapReduce History Server');
   });
+  it('Should disable standard storage option for existing runtime and have reattachable selected', async () => {
```

Comment:
please add "GCE" to this test name, and show somewhere inside the test that this is GCE

Reply:
Done